packs/feats/ancestry/lizardfolk/iruxi-armaments.json

```
@@ -63,21 +63,24 @@
                 "rollOption": "iruxi-armaments"
             },
             {
-                "key": "DamageDice",
-                "override": {
-                    "dieSize": "d6"
-                },
+                "key": "DamageAlteration",
+                "mode": "upgrade",
                 "predicate": [
-                    "iruxi-armaments:claw"
+                    "iruxi-armaments:claw",
+                    "item:tag:lizardfolk-claws"
                 ],
-                "selector": "claw-damage"
+                "property": "dice-faces",
+                "selectors": [
+                    "claw-damage"
+                ],
+                "value": 6
```

This damage alteration would affect every single damage dice, including bonus ones. For base weapon damage alteration, Item Alteration is the preferred method, like this.
```suggestion
                "itemType": "weapon",
                "key": "ItemAlteration",
                "mode": "upgrade",
                "predicate": [
                    "iruxi-armaments:claw",
                    "item:tag:lizardfolk-claws"
                ],
                "property": "damage-dice-faces"
```

---

Check when you get a moment.

Thanks!